specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimportalrevisions.json

```diff
@@ -0,0 +1,326 @@
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PortalRevision"
+        ],
+        "operationId": "PortalRevision_Get",
+        "description": "Gets developer portal revision specified by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementGetPortalRevision": {
+            "$ref": "./examples/ApiManagementGetPortalRevision.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/PortalRevisionIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Gets the specified developer portal revision entity.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PortalRevisionContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PortalRevision"
+        ],
+        "operationId": "PortalRevision_CreateOrUpdate",
+        "description": "Creates a new developer portal revision.",
+        "x-ms-examples": {
+          "ApiManagementCreatePortalRevision": {
+            "$ref": "./examples/ApiManagementCreatePortalRevision.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/PortalRevisionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PortalRevisionContract"
+            }
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
how does a user delete these?

Reply:
We do not allow to delete these items. We have the limit to 100 revisions and remove the oldest if it getting more than 100.